packages/DataGrid/src/DataGrid.styles.js

```diff
@@ -0,0 +1,258 @@
+        */
+        box-shadow: 3px 0px 0px 0px ${tokens.border.color};
+        overflow-x: hidden !important;
+        /* overflow: hidden !important; */
+        z-index: 1;
+        ::-webkit-scrollbar {
+          width: 0px;
+        }
+        ::-webkit-scrollbar-track-piece {
+          -webkit-border-radius: 6px;
+          background-color: transparent;
+        }
+      }
+    `;
+  }}
+`;
+
+export const Cell = styled.div`
+  background: ${tokens.color.white};
+  border-bottom: 1px solid ${tokens.border.color};
+  border-left: 1px solid ${tokens.border.color};
+  box-sizing: border-box;
+  display: flex;
+  font-size: 16px;
+  justify-content: flex-start;
+  line-height: 1;
+  position: relative;
+
+  &:focus {
+    outline: 1px solid transparent;
+    /* outline: 0; */
+  }
+
+  ${({ hasActiveRowShadow }) => {
+    return hasActiveRowShadow
+      ? css`
+          /* box-shadow: ${tokens.highlight.active.withBorder.insetBoxShadow}; */
+          &:after {
+            
+            content: "";
+            height: 100%;
+            left: 0;
+            position: absolute;
+            top: 0;
+            width: 3px;
+            z-index: 1;
+          }
+        `
+      : "";
+  }}
+`;
+
+export const CellHeader = styled(Cell)`
+  background: ${experimentalGrey};
+  border-bottom: 1px solid ${tokens.border.color};
+  border-left: 1px solid ${tokens.border.color};
+  color: ${tokens.color.black};
+  display: block;
+  font-weight: 600;
+  overflow: hidden;
+  padding: 8px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
+`;
+
+export const InnerCell = styled.div`
+  display: block;
+  height: 100%;
+  overflow: hidden;
+  padding: 8px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
+`;
+
+export const InnerElementType = styled.div``;
+
+export const InnerElementTypeMainGrid = styled.div``;
+
+export const OuterElementTypeMainGrid = styled.div`
+  /*
+   The arrow navigatoion is border sensible, please avoid adding a border to this styled components
+   doing it will create a incorrect navigation while using the arrow keys.
+  */
+
+  box-shadow: 0px 1px 0px 0px ${tokens.border.color}, 1px 0px 0px 0px ${tokens.border.color};
+`;
+
+export const GridCell = styled.div`
+  height: 1px;
+  left: 0;
+  overflow: hidden;
+  position: absolute;
+  top: 0;
+  whitespace: nowrap;
+  width: 1px;
+`;
+
+export const FillerTopRigth = styled.div`
+  background: ${experimentalGrey};
+  border: 1px solid ${tokens.border.color};
+  border-bottom: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  ${({ rowHeight, scrollBarWidth }) => {
+    const displayBlock = scrollBarWidth > 0 ? "" : "display: none;";
+
+    return `
+      ${displayBlock}
+      width: ${scrollBarWidth}px;
+      height: ${rowHeight}px;
+    `;
+  }}
+`;
+
+export const FillerBottomLeft = styled.div`
+  /* this is a small square filler on the right top corner of the table */
```

Comment:
Is this comment accurate? it says right top corner, but the styled component says bottom left..

Reply:
yes its, I updated the comment